cmd/celestia/rpc.go

```diff
@@ -282,3 +282,23 @@ func parseAddressFromString(addrStr string) (state.Address, error) {
 
 	return addr, nil
 }
+
+func parseSignatureForHelpstring(methodSig reflect.StructField) string {
```

Comment:
Would be nice to have a test for this func.

Reply:
Added a test, thank you for noticing